netsuite/10.0/modules/ROOT/pages/netsuite-connector-upgrade-migrate.adoc

```diff
@@ -10,96 +10,114 @@ Upgrade NetSuite Connector to version 10.0 to use the new features.
 |9.3.0 |10.0.0
 |===
 
-== Changes in This Release
+== New Features and Changes in This Release
 
-This release contains the following changes:
+This release contains the following new features:
 
-* These authentication types were removed:
-** *Login Authentication* 
-** *SSO Authentication* 
 * These operations were added: 
 ** *Get Account Governance Info* 
 ** *Get Integration Governance Info* 
+
+
+== Changed Functionality in This Release
+
+This release contains the following changed functionality:
+
+* The Login and SSO authentication types were removed. +
+If your integration is based on these authentication types, you must migrate to a Request or Token-Based authentication type.
+* Other nonfunctional changes were applied to the interface of the connector to make it more consistent with other Anypoint Connectors. +
+This means that some of the operation calls in your existing application might not initially work until some minor migration tasks are performed.
+* Unknown errors are now thrown as `ANY` rather than `UNKNOWN`. +
+Consequently, error codes `DISPOSE_ERROR`, `MAPPING_ERROR`, `CANT_READ_FILE`, and `TRIGGER` are now considered `ERROR_FAULT` errors instead of `UNKNOWN`.
 * The *Map SSO* operation was removed.
-* The *UKNOWN* error code was removed. +
-All unknown errors are now thrown under the *ERROR_FAULT* error code.
-* Many operation inputs were slightly altered, to make them more consistent with other Anypoint Connectors.
 * All exposed classes have been moved from the `org.mule.module.netsuite.extension.api` package to the `com.mulesoft.connector.netsuite.extension.api` package.
 
 [[changed_operations]]
 == Changed Operations
 
+The behavior of some operations has changed in NetSuite Connector 10.0.
+
+The following operations no longer default to `payload`:
```

Comment:
```suggestion
The following operations have new default parameter values:
```

Reply:
They don't have new defaults. They just no longer default to payload.

Reply:
The operations or the parameters?

Reply:
The parameters do not default to payload. Good catch.